src/sql/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -216,6 +216,13 @@ export class SelectBox extends vsSelectBox {
 		let selectOptions: SelectOptionItemSQL[] = SelectBox.createOptions(options);
 		this.populateOptionsDictionary(selectOptions);
 		super.setOptions(selectOptions, selected);
+		let selectedOptionIndex = this._optionsDictionary.get(this._selectedOption);
+		if (selectedOptionIndex === selected) { // Not generating an event if the same value is selected.
```

Comment:
What event? There doesn't seem to be any event fired from this code

Reply:
wrong comment - will update :)